flask/notejam/models.py

```diff
@@ -6,11 +6,12 @@
 
 from notejam import db
 
+PASSWORD_LEN = len(generate_password_hash('foo'))
 
 class User(db.Model, UserMixin):
     id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(120), unique=True)
-    password = db.Column(db.String(36))
+    password = db.Column(db.String(PASSWORD_LEN))
```

Comment:
The problem is that someday this might be over 100, and then the code will
break, 100 is kind of arbitary

On Mon, Sep 28, 2015 at 1:01 PM, Serhii Komar <notifications@github.com>
wrote:

> In flask/notejam/models.py
> <https://github.com/komarserjio/notejam/pull/47#discussion_r40535630>:
>
> >
> >  class User(db.Model, UserMixin):
> >      id = db.Column(db.Integer, primary_key=True)
> >      email = db.Column(db.String(120), unique=True)
> > -    password = db.Column(db.String(36))
> > +    password = db.Column(db.String(PASSWORD_LEN))
>
> I would prefer simpler and safer solution, just db.String(100).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/komarserjio/notejam/pull/47/files#r40535630>.
>